fudge-ratio/src/main/webapp/code/save.js

```diff
@@ -1,21 +1,6 @@
-const timeKey = "startingTime";
-const indexKey = "selectedIndex";
+const TIME_KEY = "startingTime";
+const INDEX_KEY = "selectedIndex";
+const EXPECTED_KEY = "expected";
 
-const saveSessionState = dropdown => {
```

Comment:
I created this code, and after all the modification, it really did not make sense, do I delete them and handle the responsibility to store the values to each "screen / component" just have consistent keys, that it